README.md

```diff
@@ -21,3 +21,37 @@ be available for use.
 # Running the image Locally
 After building the image, you can run it locally using the following command: <br>
 `podman run -p 8080:8080 MY-CUSTOM-IMAGE:latest start-singleuser.sh --ip="0.0.0.0" --port=8080`
+
+## How to add the image to opendatahub/JupyterHub
+
+* Once you have pushed your image to an image registry that is accessible by your JupyterHub instance,
+you can use the `ImageStream` template in this repository at `openshift/oc-imagestream.yaml`.
+
+
+```yaml
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    opendatahub.io/notebook-image: "true" # <-- setting it true makes it visible to ODH/JupyterHub
+  name: s2i-custom-notebook # <-- Update the ImageStream name here
+spec:
+  lookupPolicy:
+    local: true
+  tags:
+  - from:
+      kind: DockerImage
+      name: docker-registry.upshift.redhat.com/aicoe-notebooks/s2i-custom-notebook:latest # <-- Change to your container Image with tag
```

Comment:
Should it rather mention a public registry URL as an example?

Reply:
@harshad16 should we do quay instead?

Reply:
As this container image created by the pipeline were maintain in the internal registry due to the requirement of keeping them behind VPN. 
Using the same registry example would be more convenient for the users, i feel.
WDYT?

Reply:
surely we can point to quay.io :)

Reply:
i think this repo (and the `s2i-custom-notebook` image) should be usable independent of the pipeline

Reply:
Yeah, that is kind of my point - since it is publicly available, I'd not tie it into an internal pipeline or process

Reply:
so it would make more sense if we keep a public image link in the docs

Reply:
make sense,  we can use the quay.io.
sorry @4n4nd  for the inconvenience .